alphabase/spectral_library/base.py

```diff
@@ -581,51 +582,71 @@ def load_df_from_hdf(self, hdf_file: str, df_name: str) -> pd.DataFrame:
         """
         return self._get_hdf_to_load(hdf_file).__getattribute__(df_name).values
 
-    def save_hdf(self, hdf_file: str):
+    def save_hdf(self, hdf_file: str, save_mod_seq_in_other_df: bool = False):
```

Comment:
I think this is good. This shifts the responsibility to the writer of HDFs.
By default we have the simple case of 1) single df 2) loading modseq for old style dfs.

Reply:
Yes